lib/Resource.js

```diff
@@ -77,6 +77,8 @@ class Resource {
 			// Since the sourceMetadata object is inherited to clones, it is the only correct indicator
 			this.#sourceMetadata.contentModified = this.#sourceMetadata.contentModified || false;
 		}
+		this.#isModified = false;
```

Comment:
calling the private setter means we could put the private member assignment back to the end of the constructor like it was before isn't?

Reply:
I kept it there on purpose. The old location actually reset the `isModified` flag back to false after it had been set to true by one of the setters. So it "corrected" this flag, but not the sourceMetadata flag.

True, now it shouldn't matter. But just to make it a bit easier to break the tests by using the wrong setter again in the future, I thought I would keep it up there. Also, now it's closer to the `sourceMetadata.contentModified `flag. So hopefully they always get the same treatment.